test/new-e2e/tests/apm/docker_test.go

```diff
@@ -0,0 +1,159 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package apm
+
+import (
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/DataDog/datadog-agent/test/fakeintake/aggregator"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	awsdocker "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/docker"
+	"github.com/DataDog/test-infra-definitions/components/datadog/dockeragentparams"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/stretchr/testify/assert"
+)
+
+type DockerFakeintakeSuite struct {
+	e2e.BaseSuite[environments.DockerHost]
+}
+
+func TestDockerFakeintakeSuite(t *testing.T) {
+	devModeEnv, _ := os.LookupEnv("E2E_DEVMODE")
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
	isCI, _ := strconv.ParseBool(os.Getenv("CI"))
	if isCi {
		t.Skipf("blocked by APL-2786")
	}
	devModeEnv, _ := os.LookupEnv("E2E_DEVMODE")
```
Requires adding `strconv` and `os` to imports too